dags/DAG_url_processor.py

```diff
@@ -36,7 +37,7 @@ def extract_data(url):
         article.parse()
 
         return {
-            'published_at': article.publish_date,
+            'published_at': article.publish_date or find_date(url),
```

Comment:
If the publish_date is None, find_date(url) will be set, right? 
But what if the publish_date is empty i.e. "" and not None

Reply:
Updated. Could you check?